<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditScene</class>
 <widget class="QDialog" name="EditScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="save_scene">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>770</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save and quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preview">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>670</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview (low quality)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_object">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Object</string>
   </property>
  </widget>
  <widget class="QListWidget" name="object_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>256</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_light">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Light</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Camera</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QDoubleSpinBox" name="pos_z">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="position">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="pos_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="pos_y">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="position_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="angle_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="angle_y">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="angle_z">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="position_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FOV</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fov">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
    <property name="value">
     <double>70.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="ambient_light">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ambient Light</string>
   </property>
  </widget>
  <widget class="QLabel" name="ambient_light_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color Filter</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ambient">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="value">
    <double>0.600000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="filter">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No filter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sepia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Black &amp; White</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
